### ISSUE TITLE
DOC: Add Estimate Release Schedule

### ISSUE THREAD

**Issue by nociza** [langchain-ai/langchain#27373]
### Checklist

- [X] I added a very descriptive title to this issue.
- [X] I included a link to the documentation page I am referring to (if applicable).

### Issue with current documentation:

Hi, I wonder if it would be possible to add a rough estimate of future release schedules for reference. In particular I'm waiting for a version flag to be released so that we can use GraphRAG with Langchain without dependency version issues. If we have a set release schedule we can plan our work accordingly (waiting for the official release vs. pulling the repo and resolving ourselves).

**Comment by eyurtsev** [comment]
Which packages are blocking right now? Is this due to the tenacity dep?

**Comment by nociza** [comment]
> Which packages are blocking right now? Is this due to the tenacity dep?
Yes, I saw the PR to fix it [here](https://github.com/langchain-ai/langchain/pull/22973) last week, and was waiting for it to be released!
